server/events/project_command_runner_test.go

```diff
@@ -428,6 +428,7 @@ func TestDefaultProjectCommandRunner_ApplyDiverged(t *testing.T) {
 	tmp, cleanup := TempDir(t)
 	defer cleanup()
 	When(mockWorkingDir.GetWorkingDir(prjCtx.BaseRepo, prjCtx.Pull, prjCtx.Workspace)).ThenReturn(tmp, nil)
+	When(mockWorkingDir.HasDiverged(matchers.AnyLoggingLogger(), AnyString())).ThenReturn(true)
```

Comment:
Why we needed to add this line to a logging PR: Turns out when I updated the working dir mock to support the new logger, I changed the mocked library from always returning `true` to any call of `HasDiverged` instead of have it dependent on what the user sets. See mock_working_dir.go changes for context.